docs/reference/controls/itemscontrol.md

```diff
@@ -89,7 +115,61 @@ public class Crockery
 
 The view resizes horizontally, but content is hidden when it is too high. This control does not have a built-in scrollbar (unlike `ListBox`).
 
-<img src={ItemsControlScreenshot} alt="" />
+<img src={example01Screenshot} alt="" />
+
+## Example 02
+This example binds an observable collection of `Rectangle`s (based on the Tile VM data) to an `ItemsControl`. ItemsControl.ItemPanel is set to a `Canvas` and we use a style to position the `Rectangle` within the `Canvas`.
+
+```xml
+<ItemsControl ItemsSource="{Binding TileList}">
+  <ItemsControl.ItemsPanel>
+    <ItemsPanelTemplate>
+      <Canvas Width="50" Height="50" Background="Yellow" Margin="3"/>
+    </ItemsPanelTemplate>
+  </ItemsControl.ItemsPanel>
+  <ItemsControl.ItemTemplate>
+    <DataTemplate>
+      <Rectangle Fill="Green" Height="{Binding Size}" Width="{Binding Size}"/>
+    </DataTemplate>
+  </ItemsControl.ItemTemplate>
+  <ItemsControl.Styles>
+    <Style Selector="ContentPresenter"  x:DataType="vm:Tile">
+      <Setter Property="Canvas.Left" Value="{Binding TopX}"/>
+      <Setter Property="Canvas.Top" Value="{Binding TopY}"/>
+    </Style>
+  </ItemsControl.Styles>
+</ItemsControl>
+```
+
+```csharp title='C# View Model'
+using AvaloniaControls.Models;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace AvaloniaControls.ViewModels
+{
+    public class MainWindowViewModel : ViewModelBase
```

Comment:
I just noticed you have `ViewModelBase` here. If you don't need it, you should remove it. People are not guaranteed to have that.